crypto/bls/hdkeychain/extendedkey.go

```diff
@@ -140,11 +140,11 @@ func (k *ExtendedKey) Derive(index uint32) (*ExtendedKey, error) {
 	// child is hardened.
 	//
 	// For hardened children:
-	//   data (36 bytes) = parent_private_key (32 bytes)  || index (4 bytes)
+	//   0x00 || ser256(parentKey) || ser32(i)
```

Comment:
at comments you just written about `point G2`  which is `0x00` but didn't write about `point G1` (0x01).
do am I mistake? or needed to add some comments about `point G1`

Reply:
@amirvalhalla Good find.

I have updated the comments to enhance clarity.